configs/projects/razor-server.rb

```diff
@@ -1,59 +1,12 @@
-project "razor-server" do |proj|
+project 'razor-server' do |proj|
+  proj.conflicts "pe-razor-server"
+  proj.replaces "pe-razor-server"
+  proj.provides "pe-razor-server"
```

Comment:
Should razor-server still conflict with pe-razor-server, or is this leftover from when you were trying to make them both the same package.

Reply:
hmm. I'm not sure. I would say yes? since they're essentially doing the same thing and share a lot of stuff? But I don't actually have as much context for when/why conflicts and stuff are necessary.

Reply:
@mwaggett I think you definitely don't want provides, conflicts/replaces should be okay assuming these can't actually be installed side by side

Reply:
okay, fair, that makes sense.